charts/consul/templates/crd-tcproutes-external.yaml

```diff
@@ -1,21 +1,18 @@
 {{- if and .Values.connectInject.enabled .Values.connectInject.apiGateway.manageExternalCRDs }}
```

Comment:
Are the license file changes intended? In a few places it's getting removed and a few places it's getting added. And should it be the other license?